src/Type/Definition/FieldsTrait.php

```diff
@@ -77,14 +80,14 @@ protected function buildFieldMap($fieldsOrThunk): array
         foreach ($fields as $fieldName => $fieldConfig) {
             invariant(
                 \is_array($fieldConfig),
-                \sprintf('%s.%s field config must be an array', $this->getName(), $fieldName)
+                \sprintf('%s.%s field config must be an array', $this->name, $fieldName)
             );
 
             invariant(
                 !isset($fieldConfig['isDeprecated']),
                 \sprintf(
                     '%s.%s should provide "deprecationReason" instead of "isDeprecated".',
-                    $this->getName(),
+                    $this->name,
```

Comment:
@crisu83 Why did you change `$this->getName()` to `$this->name`? :D Plus there is no `$name` attribute in this trait.

Reply:
I fixed this.